### PR TITLE
[13.0][ADD] stock_owner_restriction: New module to restrict quantities owned.

### PR DESCRIPTION
**Pull request by sergio-teruel**
cc @Tecnativa
TT25269
ping @carlosdauden @chienandalu

### CODE REVIEW

File: stock_owner_restriction/readme/USAGE.rst [hunk 1]
```diff
@@ -0,0 +1,9 @@
+To use this module you need to:
+
+#. Go to *Inventory > Overview*
+#. Create a incoming delivery order.
```
Comment (rousseldenis) [inline stock_owner_restriction/readme/USAGE.rst:6]: I think this is not limited to that kind of operation.

But that leads to another thought, shouldn't be good feature to set a field on picking type to enable the owner check or not ?

IMHO, I would at least add the use case in description.rst.

**Comment by sergio-teruel** [comment]
@rousseldenis Add changes... More flexible module :smile:

**Review by rousseldenis — APPROVED** [review]
@sergio-teruel Thanks for this excellent module! This looks great.

Could you just improve tests? Especially using SavepointCase instead and setting environment with tracking_disable to improve test speed like:

https://github.com/OCA/stock-logistics-workflow/blob/13.0/delivery_package_default_shipping_weight/tests/test_delivery_package_default_shipping_weight.py#L12

**Comment by sergio-teruel** [comment]
@rousseldenis Thanks!!! Changes done... waiting coverage report

**Review by ioans73 — APPROVED** [review]
Functional review

**Review by chienandalu — APPROVED** [review]
Tested functionality :+1:

File: stock_owner_restriction/README.rst [hunk 2]
```diff
@@ -0,0 +1,128 @@
...
+**To establish an owner to the merchandise you need to:**
+
+#. Go to *Inventory > Overview*
+#. Create a incoming delivery order.
+#. Set an owner.
+
+**The stock with owner assigned is not available in products:**
+
+#. Go to *Inventory > Master Data > Products*
+#. Search any product which is in incoming delivery order created in previos step.
```
Comment (CasVissers-360ERP) [inline stock_owner_restriction/README.rst]: previos typo

**Comment by sergio-teruel** [comment]
@CasVissers-360ERP Changes done!!! and rebase too!!

**Pull request merged**